andrewinsoul/TravelPortal
# implement search feature

**What does this PR do?**
Implement search feature

**Description of Task to be completed?**
- Filter model based on user input
- Serialize data
**How should this be manually tested?**
- Fetch the branch
- Run seed to seed database
- Start server and pass in the endpoint `{url}/v1/flight/search-flight` with the following query param:
```departure_city
destination_city
departure_date
return_date
cabin```
- you should see filtered output
